pkg/pillar/cmd/diag/diag.go

```diff
@@ -139,11 +138,15 @@ func Run() {
 	ctx.serverNameAndPort = strings.TrimSpace(string(server))
 	ctx.serverName = strings.Split(ctx.serverNameAndPort, ":")[0]
 
+	item, ok := ctx.configItemValueMap.GlobalSettings["timer.port.timeout"]
+	if !ok {
+		item = types.ConfigItemValue{}
+	}
 	zedcloudCtx := zedcloud.ZedCloudContext{
 		DeviceNetworkStatus: ctx.DeviceNetworkStatus,
 		FailureFunc:         zedcloud.ZedCloudFailure,
 		SuccessFunc:         zedcloud.ZedCloudSuccess,
-		NetworkSendTimeout:  ctx.globalConfig.NetworkTestTimeout,
+		NetworkSendTimeout:  item.IntValue,
```

Comment:
Incorrect. NetworkSendTimeout and NetworkTestTimeout are two different variables. Looks like you did a cut and paste without looking.

Reply:
The previous code set the NetworkSendTimeout to ctx.globalConfig.NetworkTestTimeout. I think this does the same thing. Should I be setting it to NetworkSendTimeout